### DISCUSSION TITLE
Можно ли как нибудь установить перевод в Potplayer?

### DISCUSSION THREAD

**Discussion by robins13** [ilyhalight/voice-over-translation#324] [Ideas]
Било бы не плохо встроить скрипт в видеоплеер Potplayer. Искал информацию в интернете и не нашел.

**Comment by FiLinX** [comment]
как минимум можно в потплеере подключать внешнюю звуковую дорожку, а саму эту дорожку перевода, получать либо вот в браузере через меню панельки VOT, либо мозгуй как через файлы .bat или .ps1 получать её с помощью vot-cli  -  https://github.com/ilyhalight/voice-over-translation/pull/153
